.github/workflows/phylogenetic.yaml

```diff
@@ -32,13 +32,13 @@ on:
         type: string
       sequences_url:
         description: |
-          URL for a sequences.fasta.zst file.
+          URL for sequences_{serotype}.fasta.zst files.
```

Comment:
Suggestion: add these details to the input description

https://github.com/nextstrain/dengue/blob/85b512cced8e944de4179446359084719c7c5515/phylogenetic/config/config_dengue.yaml#L3